<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gererFormation</class>
 <widget class="QDialog" name="gererFormation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="Titre">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gérer une formation</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="TableUVAjouter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>UV</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidgetCategorie">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Catégorie</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre de crédits obligatoires</string>
    </property>
   </column>
  </widget>
  <widget class="QTextEdit" name="textEditFormation">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>213</width>
     <height>173</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelDecriptionFormation">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>77</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description :</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelNomFormation">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>51</y>
     <width>28</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NomFormation">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>51</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Suivant">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>129</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Suivant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Valider">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelUvformation">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter les uvs faisant partie de la formation :</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="TableUVSupprimer">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>UV</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Obligatoire</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelCondition">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Condition</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelConditionManuel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pour entrer une condition, veuillez respecter la syntaxe suivante (Cf. Documentation pour plus de détails) : par exemple pour faire des tests sur le nombre de crédit d'une catégorie d'uv : cc.getValidatedCredits(&quot;Nom_de_la_categorie&quot;)&gt;12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AjouterLigne">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter une Ligne</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>gererFormation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>gererFormation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
